tfhe/Cargo.toml

```diff
@@ -51,6 +51,9 @@ concrete-csprng = { version = "0.4.0", path = "../concrete-csprng", features = [
     "generator_fallback",
     "parallel",
 ] }
+tfhe-c-api-dynamic-buffer = { version = "0.1.0", optional = true, features = [
+    "c_api",
```

Comment:
Strange to have a c-api feature when its in the name of the crate that the libs is for c-api :thinking:

Reply:
I know, but it's generating files in the target directory and cargo was being annoying when publishing the crate, so I went for this workaround...

Reply:
otherwise it's near impossible to get the actual artifact as the out directory is somewhere in a hashed/fingerprinted directory very hard to find again

Reply:
hum ok

Reply:
basically we write the header file in the target directory and cargo complained that the project generated the file and so considered the test build as dirty